core/ecschema-editing/src/Merging/ClassMerger.ts

```diff
@@ -125,12 +125,14 @@ export class ClassMerger<TClass extends ECClass> {
     }
   }
 
+  // First pass to create missing changes
   public static async mergeChanges(context: SchemaMergeContext, classChanges: Iterable<ClassChanges>) {
```

Comment:
Consider naming these methods better so someone reading them can understand their function without reading your comments.  For example mergeStubs and mergeContents